image/daemon/containerd.go

```diff
@@ -30,7 +30,7 @@ func imageWriter(client *containerd.Client, img containerd.Image) imageSave {
 		}
 		imgOpts := archive.WithImage(client.ImageService(), ref[0])
 		manifestOpts := archive.WithManifest(img.Target())
-		platOpts := archive.WithPlatform(platforms.Default())
+		platOpts := archive.WithPlatform(platforms.OnlyStrict(platforms.DefaultSpec()))
```

Comment:
Can we use about DefaultStrict()?
https://github.com/containerd/containerd/blob/ebcf876bf7b8f1cb79c60f37af08affa135c398d/platforms/defaults.go#L25

Reply:
yes we can